data/mode.txt

```diff
@@ -1 +1 @@
-Dynamic Mode
\ No newline at end of file
+sandbox
```

Comment:
Reminder that Sandbox and Dynamic Mode are the only supported modes right now. Admins can force specific antags using the Game Panel.

Reply:
The other modes should be removed, since they're remnants of early datum code, back when factions were automatically spawned at roundstart and constrained by the /gamemode.
We accidentally had a tatorling round when testing last Friday, it didn't work so well, with only 1 of the 5 players not being an antag, and a ling also being a traitor.

Reply:
I think that gamemodes should still work to some extent as a "legacy mode", although this isn't a priority (at all)

Reply:
The easiest way to implement that would be to let admins disable Dynamic Mode's automated balancing. Currently they can increase the cooldown to 9999999 for a similar effect.